docs/neural_modeling/plot_01_current_injection.py

```diff
@@ -104,8 +104,8 @@
 
 # %%
 #
-# The dataset contains several different pynapple objects, which we discussed
-# earlier today. Let's see how these relate to the data we visualized above:
+# The dataset contains several different pynapple objects, which we will explore soon. 
+# Let's see how these relate to the data we visualized above:
```

Comment:
Do you feel like there's enough of an explanation about the pynapple objects in this notebook? In the workshops, we have some sessions before this just focusing on pynapple. I think maybe we should at least drop a link to the pynapple docs, like "(see [pynapple docs]() for more details, if interested)". or something

Reply:
Yes good idea I'll add that.